docs/swarm/services.md

```diff
@@ -255,6 +255,28 @@ $ docker service create \
 0u6a4s31ybk7yw2wyvtikmu50
 ```
 
+The `--update-max-failure-ratio` flag controls what fraction of tasks can fail
```

Comment:
This now has to be opened against the `vnext-engine` branch on https://github.com/docker/docker.github.io